examples/grpcclient/main.go

```diff
@@ -59,7 +61,10 @@ func main() {
 }
 
 func callMarketsGRPC(g *provider.GRPCClient) {
-	markets, err := g.GetMarkets(context.Background())
+	authHeader := make(map[string]string)
```

Comment:
it'd also be a good idea to make all of this part of `RPCOpts`, and include the lines when creating the GRPCProvider